### ISSUE TITLE
Add timestamp in new file name

### ISSUE THREAD

**Issue by Biont** [e-alfred/nextcloud-scanner#8]
All new scans are currently called "Scan.jpg" by default. It would be cooler if they'd have a unique ID or a timestamp so scanning multiple documents gets less tedious

**Comment by Biont** [comment]
This is taken care of in the vuejs-branch

**Comment by e-alfred** [comment]
Can we close this issue then?

**Comment by Biont** [comment]
Yeah I guess. If you are okay with the changes then we can close here

**Comment by e-alfred** [comment]
There is still a problem with the timestamp: It doesn't add a file extension to the timestamp that is generated. Also it is a bit awkward to enter anything at the end of the line because the cursor is not shown:

![image](https://user-images.githubusercontent.com/10073629/59552239-b9f7f180-8f84-11e9-923f-de284b9fc240.png)

**Comment by e-alfred** [comment]
Also, the timestamp should include the time as well because people probably do more than one scan per day. :+1:

**Comment by Biont** [comment]
Lol makes sense :stuck_out_tongue_winking_eye:
